.github/workflows/approved-for-ci-run.yml

```diff
@@ -88,15 +115,33 @@ jobs:
             Feel free to review/comment/discuss the original PR #${PR_NUMBER}.
           EOF
 
-          ALREADY_CREATED="$(gh pr --repo ${GITHUB_REPOSITORY} list --head ${BRANCH} --base main --json number --jq '.[].number')"
-          if [ -z "${ALREADY_CREATED}" ]; then
-            gh pr --repo "${GITHUB_REPOSITORY}" create --title "CI run for PR #${PR_NUMBER}" \
+          LABELS=$(gh pr view ${PR_NUMBER}  --json labels --jq '.labels.[].name' | \
```

Comment:
`--repo "${GITHUB_REPOSITORY}"` is missing for `gh pr view` call

Reply:
done